src/components/Header/Header.css

```diff
@@ -210,8 +205,8 @@
   }
 
   .upper-btn {
-    max-width: 13rem;
-    height: 2.5rem;
+    /* max-width: 13rem;
```

Comment:
Code looks good. Just remove the commented code

Reply:
Done